src/Sylius/Component/Translation/Model/AbstractTranslatable.php

```diff
@@ -19,15 +19,11 @@
 abstract class AbstractTranslatable implements TranslatableInterface
 {
     /**
-     * Translations.
-     *
      * @var TranslationInterface[]
```

Comment:
But __it's__ already coupuled with them in [`__construct()`](https://github.com/Sylius/Sylius/pull/3315/files#diff-90b0a0fcc484de0d02533a37e0903387R45)...